app/view_models/workarea/storefront/order_view_model.decorator

```diff
@@ -0,0 +1,17 @@
+module Workarea
+  decorate Storefront::OrderViewModel, with: :package_products do
+    def items
+      @items_without_bundles ||= super.reject(&:bundle?)
+    end
+
+    def bundled_items
+      @bundled_items ||=  model.bundled_items.map do |item|
+        Storefront::OrderItemViewModel.new(item)
+      end
+    end
+
+    def bundled_items_for(item_id)
+      bundled_items.select { |i| i.bundle_item_id == item_id.to_s }
```

Comment:
And here!

Reply:
Ok, I guess I could extract for view models, too.